src/mesh.c

```
@@ -389,583 +323,202 @@ int mesh_update_routing(uint16_t source, uint16_t from_node, int16_t rssi, uint8
 }
 
-/**
- * Add or update a neighbor in the neighbor table
- */
-static int mesh_add_neighbor(uint16_t node_id, uint8_t band, int16_t rssi)
-{
-    if (node_id == 0 || node_id == mesh_state.our_node_id)
-    {
-        return -1;
-    }
+static int mesh_add_neighbor(uint16_t node_id, uint8_t band, int16_t rssi) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'mesh_add_neighbor' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)